Cmdline/Action/Replace.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using log4net;
+
+namespace CKAN.CmdLine
+{
+    public class Replace : ICommand
+    {
+        private static readonly ILog log = LogManager.GetLogger(typeof(Replace));
+
+        public IUser User { get; set; }
+
+        public Replace(IUser user)
+        {
+            User = user;
+        }
+
+
+        public int RunCommand(CKAN.KSP ksp, object raw_options)
```

Comment:
What's the rationale for making a new command for this? From reading #904, it sounded like this would tie into the existing `upgrade` command as needed. If we're confident enough in the replacement mod to put it in the metadata, why would we want to trouble the user with knowing whether it's an upgrade or a replaced_by?

Reply:
I think it is sufficiently different that users should have to make a distinct choice to replace a mod rather than simply upgrading. We don't want users asking us why they have unknown mods in their list suddenly when they aren't dependencies.

Reply:
Good points. But if a user just wants to get the latest updates and doesn't particularly care if they're called XYZ or XYZReplaced, I could see it being an irritation to have to run multiple commands, every time.

What about using an optional flag to create a combination command? Something like:
`ckan upgrade --all --with-replacements`